internal/cli/publish.go

```diff
@@ -279,8 +284,8 @@ func PublishCmd(ctx context.Context, outputRefs string, archs []types.Architectu
 		for arch, img := range imgs {
 			bc.Options.WantSBOM = true
 			bc.Options.Arch = arch
-			bc.Options.TarballPath = filepath.Join(bc.Options.TempDir(), bc.Options.TarballFileName())
-			bc.Options.WorkDir = filepath.Join(workDir, arch.ToAPK())
+			bc.Options.TarballPath = archTarballs[arch]
+			bc.Options.WorkDir = archWorkdirs[arch]
```

Comment:
I don't get why these changes are necessary. The arch-specific workdir is provided by the `filepath.Join()`, so shouldn't he prior and future be identical?